lib/internal/webidl.js

```diff
@@ -275,20 +277,34 @@ function createSequenceConverter(converter) {
       const val = converter(res.value, {
         __proto__: null,
         ...opts,
-        context: `${opts.context}, index ${array.length}`,
+        context: `${opts.context}[${array.length}]`,
       });
       ArrayPrototypePush(array, val);
     };
     return array;
   };
 }
 
+// https://webidl.spec.whatwg.org/#js-interface
+function createInterfaceConverter(name, prototype) {
+  return (V, opts = kEmptyObject) => {
```

Comment:
I'd recommend replacing opts with a string argument.

Reply:
It might not be simple to do that. The `opts` in the returned converter can be used in `createSequenceConverter`, which creates a different converter based on an incoming converter. It assumes the incoming one takes `opts` as object.

TL;DR these `opts` need to be consistent across all converters.

Reply:
In undici I replaced object arguments with strings for perf, but this should be left for another PR